pkg/sidecar/server.go

```diff
@@ -79,6 +79,7 @@ func (s *server) backupHandler(w http.ResponseWriter, r *http.Request) {
 
 	w.Header().Set("Content-Type", "application/octet-stream")
 	w.Header().Set("Connection", "keep-alive")
+	w.Header().Set("Trailer", "Success")
```

Comment:
Let's prefix our trailers with `X-` so that we don't interfere with possible future common trailers. We should also move from `Status: true/false` to `Success` and `Failed`.

```suggestion
	w.Header().Set("Trailer", "X-Backup-Status")
```